core/src/test/java/ai/timefold/solver/core/testdomain/invalid/entityannotatedasproblemfact/TestdataEntityAnnotatedAsProblemFactCollectionSolution.java

```diff
@@ -25,6 +27,11 @@ public TestdataEntityAnnotatedAsProblemFactCollectionSolution() {
     }
 
     @ProblemFactCollectionProperty
+    public List<TestdataEntity> getEntitiesAsFacts() {
+        return entities;
+    }
+
+    @PlanningEntityCollectionProperty
```

Comment:
Since this is an invalid class, do we need to define `@PlanningEntityCollectionProperty`?

Reply:
Yep; otherwise it would fail (for the wrong reason); a planning solution class must have at least one @PlanningEntityProperty or @PlanningEntityCollectionProperty. This is one of the reason why the test pass despite the behaviour being incorrect (the other reasons being: missing @PlanningScore annotation and a test being accidentally duplicated).